api/functions/src/ingestor/thresholdDetection.ts

```diff
@@ -31,8 +36,22 @@ export function doThresholdDetection(rmsValue: number) {
           state.thresholdValue +
           ' met, notification being sent.'
```

Comment:
Not your fault, but should the "notification being sent" message only be printed out after passing 10 bad readings?

Reply:
Probably but thats just test code not actual production stuff so we can just leave it